### DISCUSSION TITLE
2018/02/08/1932-newspaper-article-new-beacon-on-red-butte-mountain-near-salt-lake-city-ut

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#2] [Announcements]
# 2018/02/08/1932-newspaper-article-new-beacon-on-red-butte-mountain-near-salt-lake-city-ut

http://localhost:4000/2018/02/08/1932-newspaper-article-new-beacon-on-red-butte-mountain-near-salt-lake-city-ut

**Comment by jengalas** [comment]
Posted by gaplant
May 3, 2018 at 4:04 pm

_[Imported from WordPress]_

Hi! I just stumbled onto your site from a bad youtube -> wikipedia surfing excursion…
I have very little to say other than: it looks like a great site, I’m going to be reading it for weeks, but most importantly:

the thumbnail to this article looks like they are hoisting a donkey to the top of the beacon tower 😛
Silly 1930s people- donkeys are hard to see at night, even in Utah.

**Reply by jengalas** [reply]
Ha-ha … hilarious and true! Thank you!
